app/routes.js

```diff
@@ -0,0 +1,24 @@
+var	express = require('express');
+var	config = require('./controller/settings');
+
+var	router = express.Router();
```

Comment:
This is throwing a build error:
 A function with a name starting with an uppercase letter should only be used as a constructor.